website/src/components/SequenceDetailsPage/SequencesDisplay/getSequenceNames.tsx

```diff
@@ -1,5 +1,11 @@
 import { type ReferenceGenomesSequenceNames, SINGLE_REFERENCE } from '../../../types/referencesGenomes.ts';
-import type { SequenceName } from '../../../utils/sequenceTypeHelpers.ts';
+import {
+    getMultiPathogenNucleotideSequenceNames,
+    getMultiPathogenSequenceName,
```

Comment:
I'm wondering throughout this if `SequenceName` is the right name for this. I appreciate it was an existing name but maybe its more confusing now we have to do more with it. To me the sequence name would be e.g. the accession of a sequence entry. This is more segment name or contig name or something.

Reply:
We discussed and also want to rejig `SequenceName` to be something like `SegmentInfo | GeneInfo` or equivalent - we will tackle in a future PR